code/modules/goonchat/browserOutput.dm

```diff
@@ -199,6 +199,8 @@ GLOBAL_DATUM_INIT(iconCache, /savefile, new("tmp/iconCache.sav")) //Cache of ico
 
 	if(!cookie)
 		return
+	else
+		return
```

Comment:
This else clause is redundant as the function will always return. You should remove the else clause as this is improper coding practice

Reply:
Terra dabbing on haters